docker/archimedes-base/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM python:3.8-slim
```

Comment:
This is basically all the python and binary requirements for archimedes.  I decided to fork this into a separate base image, rather than pile it into the etna-base image, simply because these requirements have less overlap and machines that eventually are running archimedes scripts likely won't be the same machines that are web servers (say, C4).